Tasks/ValidHexCode.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Tasks
+{
+    /// <summary>
+    /// Validation hex code
+    /// </summary>
+    public class ValidHexCode
+    {
+        /// <summary>
+        /// Validation of hex code
+        /// </summary>
+        /// <param name="inputHex">
+        /// input hex code
+        /// </param>
+        /// <returns></returns>
+        public static bool CheckIsHexValid(string inputHex)
+        {
+            if (inputHex.Length == 8 && inputHex.StartsWith("0x"))
```

Comment:
There is such thing as magic number. Here 8 is like it, it when you see some number and you need to guess what does it means. Create variable with meaningful name